packages/nextjs/components/Arrow_Icon.tsx

```diff
@@ -0,0 +1,22 @@
+"use client";
+
+import { useTheme } from "next-themes";
+
+const Arrow_Icon = () => {
+  const { theme } = useTheme();
+
+  return (
+    <svg width="12" height="12" viewBox="0 0 16 17" fill="none" xmlns="http://www.w3.org/2000/svg">
+      <path
+        fillRule="evenodd"
+        clipRule="evenodd"
+        d="M16 16.5V0.5H0V2.5H13L0 15.5L1 16.5L14 3.5V16.5H16ZM16 0.5H13V3.5H16V0.5Z"
+        fill={`
+                    ${theme === "light" ? "#000000" : "#868686"}
+                  `}
+      />
```

Comment:
Here's a tip: always check the browser console when running your app. When loading the page it gives the following warning:

Warning: Prop 'fill' did not match. Server: "\n #868686\n " Client: "\n #000000\n " path svg Arrow_Icon@webpack-internal:///(app-pages-browser)/./components/Arrow_Icon.tsx:9:76 a LinkComponent@webpack-internal:///(app-pages-browser)/./node_modules/next/dist/client/link.js:105:254 div div Card@webpack-internal:///(app-pages-browser)/./components/Card.tsx:10:48 div div div

It would be great if you could fix this issue, please!

Reply:
it was happening due to useTheme not being available in server-side. fixed it by waiting till the component mounts